tests/integration_tests/query_context_tests.py

```diff
@@ -728,3 +730,183 @@ def test_get_label_map(app_context, virtual_dataset_comma_in_column_value):
         "count, col2, row2": ["count", "col2, row2"],
         "count, col2, row3": ["count", "col2, row3"],
     }
+
+
+def test_time_column_with_time_grain(app_context, physical_dataset):
+    column_on_axis: AdhocColumn = {
+        "label": "I_AM_A_ORIGINAL_COLUMN",
```

Comment:
nit (don't bother changing this unless you happen to have the file open in your IDE 😆): 
```suggestion
        "label": "I_AM_AN_ORIGINAL_COLUMN",
```